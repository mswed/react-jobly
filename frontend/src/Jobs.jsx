const Jobs = ({ jobs }) => {
  return <div>List of all jobs</div>;
};

export default Jobs;
