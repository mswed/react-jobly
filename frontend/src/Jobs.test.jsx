import { it, expect } from 'vitest';
import { render } from '@testing-library/react';
import Jobs from './Jobs';

it('renders without crashing', async () => {
  render(<Jobs />);
});

it('matches snapshot', async () => {
  const { asFragment } = render(<Jobs />);

  expect(asFragment()).toMatchSnapshot();
});
