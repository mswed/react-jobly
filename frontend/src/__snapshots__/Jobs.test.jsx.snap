// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    List of all jobs
  </div>
</DocumentFragment>
`;
